bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/setup/ThingSetupManager.java

```diff
@@ -260,6 +260,12 @@ public void enableChannel(ChannelUID channelUID) {
                     }
                     item.addTags(channelType.getTags());
                     item.setCategory(channelType.getCategory());
+                    if (channelType.getState().getPattern() != null) {
+                        String label = channelType.getLabel() + " [" + channelType.getState().getPattern() + "]";
```

Comment:
I would not add the pattern here as part of the label. It should be in the responsibility of the UI to render at the end a label, which combines real label and meta data like the pattern (Paper UI should already support it). If it is needed for Sitemap, I would suggest to add it there.